justfile

```diff
@@ -0,0 +1,94 @@
+# recipes for the `just` command runner: https://just.systems
+# how to install: https://github.com/casey/just#packages
+
+# the central source of truth for Bonfire extension project boilerplate
+# is tracked at https://github.com/bonfire-networks/bonfire-extension-boilerplate
+
+# we load all vars from .env file into the env of just commands
+set dotenv-load
+# and export just vars as env vars
+set export
+
+## Main configs - override these using env vars
+
+APP_VSN_EXTRA := env_var_or_default("APP_VSN_EXTRA", "")
+DB_DOCKER_VERSION := env_var_or_default('DB_DOCKER_VERSION', "17-3.5")
+DB_DOCKER_IMAGE := env_var_or_default('DB_DOCKER_IMAGE', if arch() == "aarch64" { "ghcr.io/baosystems/postgis:"+DB_DOCKER_VERSION } else { "postgis/postgis:"+DB_DOCKER_VERSION+"-alpine" })
+DB_STARTUP_TIME := env_var_or_default("DB_STARTUP_TIME", "10")
+POSTGRES_PORT := env_var_or_default("POSTGRES_PORT", "5432")
+MIX_ENV := env_var_or_default("MIX_ENV", "test")
+POSTGRES_USER := env_var_or_default("POSTGRES_USER", "postgres")
+POSTGRES_PASSWORD := env_var_or_default("POSTGRES_PASSWORD", "postgres")
+POSTGRES_DB := env_var_or_default("POSTGRES_DB", "localhost:" + POSTGRES_PORT)
```

Comment:
ah just noticed this, POSTGRES_DB is the database name, not sure why putting host and port here?